### ISSUE TITLE
Fork/split/maintain

### ISSUE THREAD

**Issue by JOJ0** [mried/beetsplug#5]
Hi Malte,
what do you think of if I help you further maintain  these nice and usable plugins. At least the autosingleton and dirfields plugin are useful to myself and I would like to continue maintainance. I would fork them into their own separate repo (one per plugin) and try to fix them for current beets and Python versions, and make them installable. I just did a tiny fix to get dirfields running again BTW. What do you think? If you would rather keep maintaining it yourself, it's ok as well but having 5 plugins in one repo is not an easy to maintain situation IMHO, thus splitting would be required, again: Tell me what you think about all this! Thanks!

**Comment by mried** [comment]
Hi JOJ0,
thank you for you offer! Great to hear this plugins are still usefull for someone. Since I didn't use beets for a long time now, they are most likely very outdated. I don't think I'll have the time for updating them in near or even mid term. So, any help is welcome.
TL;DR: Feel free to fork and split as needed 👍

**Comment by JOJ0** [comment]
by now I only had time to split out the dirfields plugin: https://github.com/joj0/beets-dirfields

If people looking for it, get in touch and I might split out others and add installers as well. HTH
